MSAL/src/public/MSALError.h

```diff
@@ -94,8 +94,9 @@ typedef NS_ENUM(NSInteger, MSALErrorCode)
     
     MSALErrorInvalidRequest              = -42002,
     MSALErrorInvalidClient               = -42003,
-    MSALErrorInvalidGrant               = -42004,
-    MSALErrorInvalidScope               = -42005,
+    MSALErrorInvalidGrant                = -42004,
+    MSALErrorInvalidScope                = -42005,
+    MSALErrorUnhandledResponse           = -42006,
```

Comment:
would be nice to add a comment for what this error means to our developers

Reply:
done